redwood-lazylayout-compose/src/commonMain/kotlin/app/cash/redwood/lazylayout/compose/LazyListState.kt

```diff
@@ -25,6 +25,11 @@ import androidx.compose.runtime.setValue
 import app.cash.redwood.lazylayout.api.ScrollItemIndex
 
 private const val DEFAULT_PRELOAD_ITEM_COUNT = 15
+private const val SCROLL_IN_PROGRESS_PRELOAD_ITEM_COUNT = 5
+private const val PRIMARY_PRELOAD_ITEM_COUNT = 20
```

Comment:
can we make these configurable in the guest app code?

Reply:
Yes! Good idea

Reply:
I'm hoping that this will preload all of the content in money tab, since it only ever about about 14 rows. we get hitches still when scrolling and the cells below are created

Reply:
Added!